### PR TITLE
Adds quick-equip verb for drones

### LINKED ISSUE #9610: Drones can't use "E" key for quick-equip
This has come up a few times now, but I really think it deserves its own report.

The "E" key makes constructing things as a human infinitely easier, mostly because it lets you bypass some of the horrible inventory tetris and manual clicking. If you have a tool you want to put back in your belt, pressing "E" is usually the least painful way to go about doing this.

Drones, meanwhile, have to individually click each object to get it back inside their pack. This is especially bad if you carry around a second sack for materials, because it makes you have to either juggle things in hand, or drag the container icon to your sprite.

When attempting to press "E" as a drone, it now yields the message "Unrecognized or inaccessible verb: quick-equip"

### PR DESCRIPTION
**Pull request by maartenberg**
Fixes #9610.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/inventory.dm [hunk 1]
```diff
@@ -112,4 +112,31 @@
...
 
 /mob/living/simple_animal/drone/stripPanelEquip(obj/item/what, mob/who, where)
-	..(what, who, where, 1)
\ No newline at end of file
+	..(what, who, where, 1)
+
+/mob/living/simple_animal/drone/verb/quick_equip()
+	set name = "quick-equip"
+	set hidden = 1
+	if (isdrone(src))
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/inventory.dm]: don't typecast src, it's not needed.

Remove the ```if(isdrone(src))``` and the ```var/mob/living/simple_animal/drone/D = src```, replace all instances of ```D``` with ```src``` or just remove the ```D```s (Both would work, Just removing the ```D```s is preferred)

Comment (Miauw62) [inline code/modules/mob/living/simple_animal/friendly/drone/inventory.dm]: and by not needed we mean "completely fucking awful"

Comment (spasticVerbalizer) [inline code/modules/mob/living/simple_animal/friendly/drone/inventory.dm]: The typecasting thing is done in human inventory [here](https://github.com/tgstation/-tg-station/blob/master/code/modules/mob/living/carbon/human/inventory.dm#L162), which is why I thought there was a reason for it being done that way

**Comment by buttsmcgee** [comment]
marry me

**Comment by Cheridan** [comment]
looks copypasty :O

**Comment by spasticVerbalizer** [comment]
Sorry ;__;

**Comment by spasticVerbalizer** [comment]
It could be less copypasty if there was a way to get all the equip slots on a mob, but _as far as I know_ something like that does not exist because of the way inventory is done

**Comment by Cheridan** [comment]
If there's no other feasible way then this will have to suffice

**Pull request merged**